production.Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN apk --update --no-cache add \
     "make~=4.3" \
     "nodejs~=14" \
     "npm~=7" \
+    "curl~=7" \
```

Comment:
This is not needed and it is not the image used by PostHog Cloud. See https://github.com/PostHog/posthog-cloud/blob/master/prod.web.Dockerfile

Reply:
hmm right - so curl is there on the cloud image

Reply:
found the issue: we install then uninstall curl on cloud

Reply:
https://github.com/PostHog/posthog-cloud/pull/171